README.md

```diff
@@ -55,7 +55,7 @@ Your skills contribution pull requests must include the following:
 
 Taxonomy skill files must be a valid [YAML](https://yaml.org/) file named `qna.yaml`. Each `qna.yaml` files contains a set of key/value entries with the following keys:
 
-- `task_description`: A description of the skill. **Required**
+- `task_description`: A description of the skill. This information is very critical and should be accurate. You should be able to mention `precise` to get precise generated data **Required**
```

Comment:
I don't think this is true. I thought `task_description` was just for humans to understand what this was for, nothing to do with the sdg?

Reply:
This is not true. The task_description is part of the prompt to the teacher model, so it is very important. I need to propose some improved wording here.

Reply:
Ah! maybe i crossed the wire with `domain` then? Thanks for the clarification.

Reply:
Well it turns out domain is important too (but for knowledge). So we need to make improvements to that part of the readme too.